recipes/sdl_mixer/cmake/conanfile.py

```diff
@@ -0,0 +1,212 @@
+import os
```

Comment:
How much of the recipe itself did you change? Is it worth the file split?

Reply:
The project changed its CMake config and dependencies substantially since the current v2.0.4.
Cramming it all together seemed like a worse option with no real upside.
Here's a PR to migrate the existing recipe: #21782